### ISSUE TITLE
Error in Number_types 

### ISSUE THREAD

**Issue by maxGimeno** [CGAL/cgal#4575]
## Issue Details

The test file 'eigen.cpp'  in Number_types  is not tested, but if it were, there would be an error on MSVC 2015  saying:
```
'dummy_precision': is not a member of 'Eigen::NumTraits<CGAL::Sqrt_extension<double,double,CGAL::Tag_false,CGAL::Tag_false>>'
```

See [here](https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-5.1-Ic-95/Number_types/TestReport_afabri_x64_Cygwin-Windows10_MSVC2015-Debug-64bits.gz) for full information.
## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): Windows
* Compiler: MSVC2015
* Release or debug mode: Debug
* Specific flags used (if any):  /DWIN32 /D_WINDOWS /W3 /GR /EHsc -D_CRT_SECURE_NO_DEPRECATE -D_SCL_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS /fp:strict /fp:except- /wd4503 /bigobj /MDd /Ob0 /Od /RTC1  /machine:x64 /DEBUG:NONE /INCREMENTAL:NO

* CGAL version: Probably 4.14
* Other libraries versions if used (Eigen, TBB, etc.): Eigen 3.3.4

**Comment by sloriot** [comment]
@danston  is this issue fixed by  #6049?

**Comment by danston** [comment]
The error fixed in #6049 was different but I have not recently encountered this error in the test suite. Moreover, the platform where this error showed up is not a part of the test suite anymore. I checked Christo MSVC 2015 and it does not seem to show this error in the recent test results.
